Sources/MySQL/Protocol/MySQLBinaryResultsetRow.swift

```diff
@@ -93,6 +93,10 @@ struct MySQLBinaryResultsetRow {
                             second: bytes.requireInteger(endianness: .little),
                             microsecond: 0
                         )
+
+                        if length == 8 {
+                            let _: UInt8 = try bytes.requireInteger(endianness: .little)
```

Comment:
Is it possible this is throwing away time information or is this always `0`? Maybe we should add an assertion?

Reply:
to check the type of information? and the value of the data? to add as microsecond or not?

Reply:
Yes, this PR is always passing `0` for microsecond even when `length == 8`. I'm wondering if that's correct since there could be information in the last byte being discarded here.